packages/core/src/DreifussWysiwygEditor.tsx

```diff
@@ -171,124 +172,126 @@ function DreifussEditor(props: DreifussWysiwygEditorOptions) {
   }, [charCount])
 
   return (
-    <Plate
-      id={id}
-      onChange={(val: TNode[]) => props.onChange(handleOnChange(val))}
-      plugins={plugins(enablePlugins, {findReplace: findReplacePlugin})}
-      components={components}
-      options={options}
-      editableProps={editableProps as EditableProps}
-      initialValue={JSON.parse(
-        JSON.stringify(value?.map(block => ({...block, id: Math.random()})))
-      )}>
-      <ToolbarBalloon editor={editorRef} />
-      {!displayOnly && (
-        <HeadingToolbar>
-          {enablePlugins.basicElements && <ToolbarBasicElementsButtons editor={editorRef} />}
+    <div className="dreifuss-wrapper">
```

Comment:
what is the reason to put `<Plate>` inside the div

Reply:
I wanted to set font-family for the plate, and thinking of moving the styling related to `displayOneLine` in the css file as well. what do you think?